### ISSUE TITLE
Crashes when scaling terrain to 0

### ISSUE THREAD

**Issue by Skintillion** [godotengine/godot#32405]
**Godot version:**
3.2.alpha.colinou.edf9055b7

**OS/device including version:**
Windows 7 64 bit

**Issue description:**
Simple scene, with box mesh terrain and texture. When use mouse to transform, if I go below 0 engine crashes. Says driver error. If I go quickly and it doesn't hit exactly 0 it's fine.

![image](https://user-images.githubusercontent.com/1521506/65809390-ead5ee00-e151-11e9-8288-d43562752bbe.png)

![image](https://user-images.githubusercontent.com/1521506/65809394-f75a4680-e151-11e9-96a3-84677e5895f7.png)

**Steps to reproduce:**
Scale simple mesh to 0 height

**Minimal reproduction project:**
Box mesh in a 3d Scene.

![image](https://user-images.githubusercontent.com/1521506/65809426-37b9c480-e152-11e9-98e6-a30110bd3800.png)

**Comment by fire** [comment]
Anytime the scale is 0, the bullet physics subsystem crashes.

**Comment by qarmin** [comment]
For me it only print error to terminal:
```
ERROR: orthonormalize: Condition ' determinant() == 0 ' is true.
   At: core/math/basis.cpp:81.
ERROR: invert: Condition ' det == 0 ' is true.
   At: core/math/basis.cpp:69.
```

**Comment by hbina** [comment]
@fire should it be bounded to never go to 0 then?

**Comment by Anutrix** [comment]
Similar to https://github.com/godotengine/godot/issues/14864.  Scale should never go to zero for anything.

**Comment by madmiraal** [comment]
Fixed with #41426, cherry-picked for 3.2.4.
